### ISSUE TITLE
CoCED uses the wrong flag to count explorations in the COC revamp mod

### ISSUE THREAD

**Issue by bqbum** [tmedwards/CoCEd#19]
I originally thought the problem was with the mod, but after I filed an issue report someone dug around and found the heart of the problem. Link to the mods issue report https://github.com/Kitteh6660/Corruption-of-Champions-Mod/issues/604 

My understanding is the editor and mod are using different variables to determine exploration counters, but its mostly over my head as I have only very basic programing skills and haven't even seen the source code. In any case I was asked to file an issue here so refer to the link above.

**Comment by tmedwards** [comment]
Per the [changelog for the current CoCEd release](https://github.com/tmedwards/CoCEd/releases "https://github.com/tmedwards/CoCEd/releases"):
> * Updated for CoC-Revamp-Mod v1.3.25.

**TL;DR:** CoCEd has not been updated to work with that version of CoC-Revamp,

Before you ask, I have no ETA on when it will.  When I last touched the code, I was in the middle of adding specific support for CoC-Xianxia.  Life has intervened and I haven't had much time to work on CoCEd lately.

**Comment by alm367** [comment]
No rush or anything, just curiosity, is there any chance you could return to this project sense the revamp mod is now at v1.4.12

**Comment by tmedwards** [comment]
It's probably not perfect, but a [new release (v1.3.2)](https://github.com/tmedwards/CoCEd/releases/latest) was just made.

After this long, I have no idea what is/isn't working as intended with the Revamp mod, so new issues should be raised if necessary.
